src/nvim/screen.c

```diff
@@ -2116,14 +2122,15 @@ static int win_line(win_T *wp, linenr_T lnum, int startrow, int endrow, bool noc
   int right_curline_col = 0;
 
   // draw_state: items that are drawn in sequence:
-#define WL_START        0               // nothing done yet
-# define WL_CMDLINE     WL_START + 1    // cmdline window column
-# define WL_FOLD        WL_CMDLINE + 1  // 'foldcolumn'
-# define WL_SIGN        WL_FOLD + 1     // column for signs
-#define WL_NR           WL_SIGN + 1     // line number
-# define WL_BRI         WL_NR + 1       // 'breakindent'
-# define WL_SBR         WL_BRI + 1       // 'showbreak' or 'diff'
-#define WL_LINE         WL_SBR + 1      // text in the line
+#define WL_START        0                 // nothing done yet
+# define WL_CMDLINE     (WL_START + 1)    // cmdline window column
```

Comment:
while we are here, the space in `# define` should be removed in all of these cases. It's an accident from the original Vim source import.

Reply:
`uncrustify` can fix this. There is a rule that specifically makes the formatting exactly like the style guide (one additional space after # for each level, no space before #).

As for why I haven't fixed it yet...you never asked ;)

Reply:
I specifically avoided changing that because I knew it's better to fix it in the uncrustify rules :)